FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Shashank Mishra on 25/04/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageMapper {
+	static private var OK_200 = 200
+
+	private struct Root: Decodable {
+		let items: [Item]
+		public var feedImages: [FeedImage] {
```

Comment:
The `public` keyword can be omitted since `Root` has a private access level. Declaring a property as `public` has no effect.

```suggestion
		var feedImages: [FeedImage] {
```

Reply:
Updated